runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorker.java

```diff
@@ -1892,6 +1891,14 @@ private void invalidateStuckCommits() {
     }
   }
 
+  public static String constructWorkId(Windmill.WorkItem workItem) {
```

Comment:
How about letting this be in `Work.java`? and then it can be called like `Work.getId` or `Work.getLatencyTrackingId` etc.

Reply:
cf55f3f7ca1c0db5f22c32f4149fb639b5da9009